libshaderc/include/shaderc.h

```diff
@@ -63,6 +63,15 @@ typedef enum {
   shaderc_profile_es,
 } shaderc_profile;
 
+// Used in the result module (shaderc_spv_module) to tell the type of errors
+// generated during compilation.
+typedef enum {
+  shaderc_compilation_result_success,
```

Comment:
explicitly set this to 0.  It's a useful pattern to have success always be 0, and error cases non-zero just like exit status from a process.
Declaring it explcitly to 0 means callers can rely on this behaviour.